Project-Euler/Problem007.js

```diff
@@ -0,0 +1,27 @@
+import { PrimeCheck } from '../Maths/PrimeCheck.js'
+
+/**
+ * Find nth Prime Number
+ *
+ * P.S.(Project Euler - 007):
+ * By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13.
+ * What is the 10001st prime number?
+ *
+ * @param {Number} n
+ * @returns {Number} returns the nth prime number
+ */
+export const nthPrime = (n) => {
+  if (n < 1) {
+    return 'Invalid Input'
+  }
+
+  let count = 0
+  let inc = 2
+  while (count < n) {
+    if (PrimeCheck(inc)) {
+      count++
+    }
+    inc++
+  }
+  return inc - 1
```

Comment:
If you increment at the beginning of the loop, you don't have to decrement here

Reply:
Sorry @appgurueu  but again I got confused. Incrementing at the beginning will also do the same at the end of the loop. In both cases, we have to decrease the value.

Reply:
No. If you increment at the beginning of the loop (and start with `candidate = 1`), when `PrimeCheck(candidate)` being truthy triggers `count++`, you don't enter the next iteration and thus `candidate` holds the correct number rather than the last prime number plus one.

Reply:
My bad! Got your point. I will make the changes.
Definitely, there're many ways to solve the it.